### ISSUE TITLE
NuGet package for foundation-sites is missing scss/_vendor folder

### ISSUE THREAD

**Issue by markau** [zurb/foundation-sites#9790]
#### How to reproduce this bug:

1. Use NuGet package manager to install foundation-sites
2. Look in the scss folder and note absence of a _vendor subfolder

#### What should happen:

The scss folder should contain a _vendor subfolder

#### What happened instead:

There is no _vendor subfolder

**Comment by dragthor** [comment]
I am not seeing this problem.  Did you run anything (npm start or npm test:javascript or test:css) ?

![capture](https://cloud.githubusercontent.com/assets/156634/23086209/c8d435fc-f539-11e6-9b87-b2afdbf38559.PNG)

**Comment by IamManchanda** [comment]
@kball please close this as it seems **Invalid** to me

**Comment by markau** [comment]
Sorry to raise the issue and not respond @dragthor, @IamManchanda, @kball 

I just reproduced the issue.  Here are the steps:

- Visual Studio 2015 update 3
- Windows 10 x64
- Create new ASP.NET MVC solution (.NET framework - i.e. v4.5.2) 
- Open Nuget Package Manager for solution
- Search for foundation 
- Install this package:

![image](https://cloud.githubusercontent.com/assets/3004849/23503443/bf5dd80a-ff8b-11e6-9a8b-ad9dba71d328.png)

And there's no `_vendor` folder:

![image](https://cloud.githubusercontent.com/assets/3004849/23503465/d37bedae-ff8b-11e6-9021-e110040edabd.png)

**Comment by dragthor** [comment]
I will look into this when I get into the office. I have a similar
environment.

**Comment by dragthor** [comment]
@markau You are correct the `_vendor` folder does not appear.  Even when I manually add the folder, SASS things still do not work correctly - even with VS.NET Web Essentials.  Sorry, there is not a decent integrated experience with Visual Studio.  And its not specific to Zurb Foundation.

We have always built & used Foundation outside (cmd line and/or VS Code) of Visual Studio, then used the output within our projects.

**Comment by markau** [comment]
It was the only issue I have faced - once I got over the confusion and realised the problem, I downloaded the `_vendor` folder from the repo and everything worked from there: I am building foundation.css from sass with `node-sass` via `npm` script. Sweet!

I don't know how nuget packages are built - it might just be an oversight in there.

**Comment by ricardo-lewis** [comment]
@markau How did you download the _vendor folder from this repo? It's not there.

**Comment by markau** [comment]
@ricardo-lewis, I updated my comment with the link for others' reference. Cheers.

**Comment by kball** [comment]
CC @ncoden and @abdullahsalem we need to figure out/fix this `_vendor` issue certainly for 6.4

**Comment by adrtbks** [comment]
I've got the same issue. Everything builds fine using the (manual) workaround.

**Comment by colin-marshall** [comment]
@ncoden @abdullahsalem are either of you able to look into this?

**Comment by DanielRuf** [comment]
Do we still have to check this? Who manages the NuGet package?

**Comment by DanielRuf** [comment]
I see the folder for 6.5.3 there:

<img width="274" alt="Bildschirmfoto 2019-10-19 um 17 39 14" src="https://user-images.githubusercontent.com/827205/67147705-79dcae80-f297-11e9-8748-e62f57ca1111.png">

https://www.nuget.org/api/v2/package/foundation-sites/6.5.3

And it seems the same files are also in `contentFiles`
